gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Channelizer.java

```diff
@@ -143,18 +152,18 @@ protected void initChannel(final SocketChannel socketChannel) throws Exception {
     /**
      * WebSocket {@link Channelizer} implementation.
      */
-    public final class WebSocketChannelizer extends AbstractChannelizer {
+    final class WebSocketChannelizer extends AbstractChannelizer {
```

Comment:
Small breaking change here. I'm not aware of any users which would be impacted by this but I would prefer to defer such a change to the next major release.